submissions/jimohKafayat/HTML/{137}productpage.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+	<meta charset="utf-8">
+	<title> Kahf's Online Clothing Store</title>
+</head>
+<body>
+	<h2><em>KAHF KOLLECTIONS</e></h2>
+
+
+<nav>
+	<p>CATEGORIES</p>
+	<ol>
+		<li><a href="#">Men Jalabiya</a></li>
+		<li><a href="#">Dubai Wears</a></li>
+		<li><a href="#">Women Jalabiya</a></li>
+		<li><a href="#">Ladies Gowns</a></li>
+		<li><a href="#">Corporate Wears</a></li>
+		<li><a href="#">Egyptian Gowns</a></li>
+		<li><a href="#">Veils/Hijab</a></li>
+		<li><a href="#">Islamic Kaftans</a></li>
+	</ol>
+	<img src="dufuna/images/myimage.PNG" width="400" height="500">
```

Comment:
These images used here suppose to be part of your PR. Place these images your `HTML` folder